src/modeler/AutoRest.Swagger/Validation/Extensions/NextLinkPropertyMustExist.cs

```diff
@@ -54,7 +54,7 @@ public override bool IsValid(object pageable, RuleContext context, out object[]
         /// <remarks>
         /// This may contain placeholders '{0}' for parameterized messages.
         /// </remarks>
-        public override string MessageTemplate => "The property '{0}' specified by nextLinkName does not exist in the 200 response schema.";
+        public override string MessageTemplate => "The property '{0}' specified by nextLinkName does not exist in the 200 response schema. \nFor the method to return an IEnumerable, set nextLinkName to null. \nFor a method to return a chained object, set nextLinkName to appropriate field in the list object.";
```

Comment:
should we take a text extract from https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable ? Maybe something like: "The property '{0}' specified by nextLinkName does not exist in the 200 response schema. \nPlease, specify the name of the property that provides the nextLink. If the model does not have the nextLink property then specify null"
Even though this does not include the full explanation, we could have the rest in the documentation.

Also, does the rule need any update to allow a "null" value for this property?